azure/Kqlmagic/kql_magic.py

```diff
@@ -135,6 +135,10 @@ class Kqlmagic(Magics, Configurable):
         "the kql connection will use the cloud as specified "
     )
 
+
+    enable_sso = Bool(True, config = True, help="enbale or disable SSO. if enabled, SSO will only work if the environment parameter KQLMAGIC_SSO_ENCRYPTION_KEYS is set properly")
+    
+    token_cleanup_time = Int(24, config=True,help= "specifies the time for a token to expire (to be deleted later, if SSO is enabled), in hours. default is one day.")
```

Comment:
1. user don't really undertsna what token is, so pls rename to sso_cleanup_interval.
2. becuase the token change only on save (and not on restore) lets change the default to 1 week.
3. help: specifies the time interval in hours, since last update of sso cache, that triggers cleanup of the cache.